docs/developer_guide/build.md

```diff
@@ -84,7 +84,7 @@ The docker image built from previous step has embedded some important build info
 these info with docker `inspect` command.
 
 ```
-docker inspect apache/yunikorn:scheduler-latest
+docker inspect apache/yunikorn:scheduler-amd64-latest
```

Comment:
Please mention that this is an architecture specific value. The default depends on the HW used. ARM based machines: arm64v8, Intel/AMD: amd64